engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/compiler/StubbedFragmentSourceDefinitionPreparer.scala

```diff
@@ -82,6 +87,13 @@ class StubbedFragmentSourceDefinitionPreparer(
             Valid(context)
           }
         }
+
+      // TODO local: question - either this is unimplemented or traits like CustomizableContextInitializerSource cant
```

Comment:
What about this question? Why now is necessary to have such a dumb method?

Reply:
whoops, forgot about this. I thought it would be a more consistent to have the new interfaces around `FlinkSource` have self bound `FlinkSource` instead of `Source` . This is the only place where it caused the problem and this source is a stub with other unimplemented method `testRecordParser`. I can relax the self type from `FlinkSource` to `Source` and this won't be needed. WDYT?

Reply:
ok

Reply:
done, relaxed the type and its not needed anymore :+1: